packages/migration/src/migrations/hearth/20230602120944-populate-valid-locationid.ts

```diff
@@ -150,10 +151,10 @@ async function updateEventLocationIdOrJurisdictionIds(db: Db, elasticDoc: any) {
     )
```

Comment:
Shouldn't this be similar to `packages/migration/src/migrations/hearth/20221110092434-add_identifiers_DOB_gender_location.ts`?